internal/compiler/generator/rust.rs

```diff
@@ -2929,6 +2929,10 @@ fn compile_builtin_function_call(
             quote!(#(#a)*.as_str().parse::<f64>().unwrap_or_default())
         }
         BuiltinFunction::StringIsFloat => quote!(#(#a)*.as_str().parse::<f64>().is_ok()),
+        BuiltinFunction::StringIsEmpty => quote!(#(#a)*.is_empty()),
+        BuiltinFunction::StringCharacterCount => {
+            quote!( slint::private_unstable_api::re_exports::UnicodeSegmentation::graphemes(#(#a)*.as_str(), true).count() as i32 )
```

Comment:
```suggestion
            quote!( sp::UnicodeSegmentation::graphemes(#(#a)*.as_str(), true).count() as i32 )
```

`sp` is an alias for `slint::private_unstable_api::re_exports` which is used everywhere in the generated code.